behaviour/graql/language/match.feature

```diff
@@ -1020,6 +1335,20 @@ Feature: Graql Match Clause
       | PER |
 
 
+  Scenario: comparing unbound variables throws an error
```

Comment:
this is "value comparing" right?

Reply:
Renamed to `Scenario: value comparison of unbound variables throws an error`